README.md

```diff
@@ -413,41 +413,45 @@ You can now make changes in the repository.
 
 ### Building the AWS JDBC Driver for MySQL
 
-To build the AWS JDBC Driver without running the tests, navigate into the aws-mysql-jdbc directory and run the following command:
+To build the AWS JDBC Driver, navigate into the aws-mysql-jdbc directory and run the following command:
 
 ```bash
-gradlew build -x test
-```
-
-To build the driver and run the tests, you must first install [Docker](https://docs.docker.com/get-docker/).  After installing Docker, execute the following commands to create the Docker servers that the tests will run against:
-
-```bash
-$ cd aws-mysql-jdbc/docker
-$ docker-compose up -d
-$ cd ../
+gradlew build
 ```
 
-Then, to build the driver, run the following command:
+To build the driver without running the tests, run the following command:
 
 ```bash
-gradlew build
+gradlew build -x test
 ```
 
 ### Running the Tests
 
-After building the driver, and installing and configuring Docker, you can run the tests in the ```aws-mysql-jdbc``` directory with the following command:
+To run the tests, you must first install [Docker](https://docs.docker.com/get-docker/). After building the driver, and installing and configuring Docker, you can run the tests in the ```aws-mysql-jdbc``` directory with the following command:
 
 ```bash
 gradlew test
 ```
 
-To shut down the Docker servers when you have finished testing:
+To run only the community tests, use the following command:
+```bash
+gradlew test-community-docker
+```
 
+To run only the integration tests, you will need some environment variables and an AWS Aurora database. A description of the necessary environment variables is in the table below. To run the integration tests, use the following command (replace the <> tags with the appropriate values):
 ```bash
-$ cd aws-mysql-jdbc/docker
-$ docker-compose down && docker-compose rm
-$ cd ../
+DB_READONLY_CONN_STR_SUFFIX=<.cluster-ro-abcdefghi.us-east-1.rds.amazonaws.com> TEST_USERNAME=<username> TEST_PASSWORD=<password> TEST_DB_CLUSTER_IDENTIFIER=<db-mysql> DB_CONN_STR_SUFFIX=<.abcdefghi.us-east-1.rds.amazonaws.com> TEST_DB_USER=<jane_doe> gradlew test-integration-docker
```

Comment:
Documentation already have references to AWS domain name. I think it's better to be consistent and use them.
Previous references:
db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com